src/library/Main.java

```diff
@@ -205,72 +212,70 @@ private static void ADD_ITEM() {
 
         while (itemType == null) {
             String type = input(typeMenu);
-            
+
             switch (type.toUpperCase()) {
-            case "B": 
-                itemType = ItemType.BOOK;
-                break;
-                
-            case "D": 
-                itemType = ItemType.DVD;
-                break;
-                
-            case "V": 
-                itemType = ItemType.VHS;
-                break;
-                
-            case "C": 
-                itemType = ItemType.CD;
-                break;
-                
-            case "A": 
-                itemType = ItemType.CASSETTE;
-                break;
-                
-            case "": 
-                return;
-            
-            default:
-                output(type + " is not a recognised Item type");
-    
+                case "B":
+                    itemType = ItemType.BOOK;
+                    break;
+
+                case "D":
+                    itemType = ItemType.DVD;
+                    break;
+
+                case "V":
+                    itemType = ItemType.VHS;
+                    break;
+
+                case "C":
+                    itemType = ItemType.CD;
+                    break;
+
+                case "A":
+                    itemType = ItemType.CASSETTE;
+                    break;
+
+                case "":
+                    return;
+
+                default:
+                    output(type + " is not a recognised Item type");
+
             }
         }
 
-        String AuThOr = input("Enter author: ");
-        String TiTlE  = input("Enter title: ");
-        String CaLl_NuMbEr = input("Enter call number: ");
-        Item BoOk = LIBRARY.addItem(AuThOr, TiTlE, CaLl_NuMbEr, itemType);
-        output("\n" + BoOk + "\n");
-        
+        String author = input("Enter author: ");
+        String title = input("Enter title: ");
+        String callNumber = input("Enter call number: ");
+        Item book = LIBRARY.addItem(author, title, callNumber, itemType);
+        output("\n" + book + "\n");
     }
 
-    
-    private static void ADD_PATRON() {
+
+    private static void addPatron() {
         try {
-            String FiRsT_NaMe  = input("Enter first name: ");
-            String LaSt_NaMe = input("Enter last name: ");
-            String EmAiL_AdDrEsS = input("Enter email address: ");
-            long PhOnE_NuMbEr = Long.valueOf(input("Enter phone number: ")).intValue();
-            Patron PaTrOn = LIBRARY.addPatron(FiRsT_NaMe, LaSt_NaMe, EmAiL_AdDrEsS, PhOnE_NuMbEr);
-            output("\n" + PaTrOn + "\n");
-            
-        } catch (NumberFormatException e) {
-             output("\nInvalid phone number\n");
+            String firstName = input("Enter first name: ");
+            String lastNae = input("Enter last name: ");
```

Comment:
#PR43 - D1 : typo in variable name. Please rename to 'lastName'

Reply:
https://github.com/elliotmitchum/ITC205/pull/43/commits/d9135d6585e083ab375330a2c76a02385a4740b8